design/one-pager-trait-workload-interaction-mechanism.md

```diff
@@ -122,9 +122,25 @@ The overall idea is for the applicationConfiguration controller to fill critical
 in the workload and trait CR it emits. In addition, we will provide a helper library so that
 trait controller developers can locate the resources they need with a simple function call. 
 Here is the list of changes that we propose.
-1. ApplicationConfig controller no longer assumes that all `trait` CRDs contain a "spec
+1. Add an optional field called `nativeTrait` to the `traitDefinition` schema. This is for the trait
```

Comment:
nativeTrait is an ambiguous name.
It is very hard for a user to correlate "native" to the implementation of OAM trait/workload interaction mechanism.

Reply:
any suggestions?